tests/doctor/test_doctor.test_unexpected_exception.approved.txt

```diff
@@ -30,10 +30,8 @@ OS: Darwin-other-system-info
 docker: 20.10.21
 docker compose: 2.12.2
 git: 2.37.1
-python: Unexpected error checking dependency
-  RuntimeError: OH NO
-python3: Unexpected error checking dependency
-  RuntimeError: OH NO
+python: 3.10.0f (location: unknown)
```

Comment:
I don't understand this, if there is an exception how can it know the version?

Reply:
Yeah I kept the test name but maybe I should change it. It's really just a failure in locating the binary with `which`.

Reply:
Test name + approval file name updated